src/app/reusable/inverted-tabs/inverted-tabs.component.scss

```
@@ -0,0 +1,40 @@
+@import "./../theme/base.scss";
+
+.tabs-below {
+    border-bottom-width: 0px;
+    border-top: 1px solid $color-grey-de;
+  }
+  
+  .tabs-below .nav-link {
```

move to parent selector.

---

those are bootstrap classes if we move to parent selector it may fail sometimes in loading
```suggestion
  .tabs-below .nav-link {
```